beta-src/src/components/ui/WDPress.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { Box } from "@mui/material";
+import Device from "../../enums/Device";
+
+interface WDPressProps {
+  children: React.ReactNode;
+  device: Device;
+}
+
+const WDPress: React.FC<WDPressProps> = function ({
+  children,
+  device,
```

Comment:
do you need to pass device?

Reply:
right now, the countrytable's layout is adjusted by different device, so the maxwidth is also  changed. the device is passed here for mobil responsive layout